### PR TITLE
Add audio dependency to tests

### PR DESCRIPTION
**Pull request by LysandreJik**
Adds the audio dependency for tests as it is now required to download audio datasets.

### CODE REVIEW

**Comment by muellerzr** [comment]
Might need sudo? 

> The Linux and macOS virtual machines both run using passwordless sudo. When you need to execute commands or install tools that require more privileges than the current user, you can use sudo without needing to provide a password.

File: .github/workflows/python-tests.yml [hunk 1]
```diff
@@ -37,6 +37,9 @@ jobs:
           pip install --upgrade pip
           pip install .[testing]
 
+          apt install -y libsndfile1-dev
```
Comment (LysandreJik) [inline .github/workflows/python-tests.yml:40]: Indeed, got too used to the self-hosted machines

```suggestion
          sudo apt install -y libsndfile1-dev
```

**Comment by LysandreJik** [comment]
Still a failure related to the audio test of the inference API cc @osanseviero

Will take a look at it this afternoon

**Comment by osanseviero** [comment]
From the error you should add `soundfile` to the testing dependencies our add `soundfile` to `datasets` `audio` installation https://github.com/huggingface/datasets/blob/master/setup.py#L104.

**Comment by osanseviero** [comment]
Thanks for this PR!

**Pull request merged**